mmv1/third_party/terraform/resources/resource_compute_instance_group_manager.go.erb

```diff
@@ -25,8 +25,8 @@ func resourceComputeInstanceGroupManager() *schema.Resource {
 			State: resourceInstanceGroupManagerStateImporter,
 		},
 		Timeouts: &schema.ResourceTimeout{
-			Create: schema.DefaultTimeout(5 * time.Minute),
-			Update: schema.DefaultTimeout(5 * time.Minute),
+			Create: schema.DefaultTimeout(15 * time.Minute),
```

Comment:
Make sure to fix the docs too! https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_instance_group_manager#timeouts

Reply:
done !